libvmi/driver/xen/xen.c

```diff
@@ -835,6 +836,26 @@ xen_init_events(
 }
 
 status_t
+xen_init_altp2m (
```

Comment:
I followed the given scheme, because I think that maybe there should be a flag there.
But switching it to void is fine with me. Any user will see the dbprint from driver_wrapper.h if he tries to use the functions with the wrong xen-version.